layouts/community/ortho_4x12/callum/config.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#define XXX KC_NO
+
+#define LAYOUT_callum(                                          \
+    L00, L01, L02, L03, L04,                                    \
+    L10, L11, L12, L13, L14,                                    \
+    L20, L21, L22, L23, L24,                                    \
+                   LT0, LT1,                                    \
+                                                                \
+    R00, R01, R02, R03, R04,                                    \
+    R10, R11, R12, R13, R14,                                    \
+    R20, R21, R22, R23, R24,                                    \
+    RT0, RT1                                                    \
+)                                                               \
```

Comment:
This bugs me, to be honest. 

```suggestion
#define LAYOUT_callum(                                          \
    L00, L01, L02, L03, L04,        R00, R01, R02, R03, R04,    \
    L10, L11, L12, L13, L14,        R10, R11, R12, R13, R14,    \
    L20, L21, L22, L23, L24,        R20, R21, R22, R23, R24,    \
                   LT0, LT1,        RT0, RT1                    \
)                                                               \
```

Reply:
Hah fair enough. I was trying to find a way to keep `callum.c` under 80 chars wide, but I admit it's probably not worth the break in consistency with how every other layout macro is arranged... Changed it back :)